Update index.md (#69663)
grafana/grafana: docs/sources/setup-grafana/configure-security/configure-authentication/okta/index.md

1. Log in to the [Okta portal](https://login.okta.com/).

> 2. Go to Admin and then select **Developer Console**.

Reviewer: In Markdown, all numbered steps are represented as `1.`

> 2. Go to Admin and then select **Developer Console**.

Reviewer: ```suggestion
1. Go to Admin and then select **Developer Console**.
```

> 3. Select **Applications**, then **Add Application**.

Reviewer: ```suggestion
1. Select **Applications**, then **Add Application**.
```

> 4. Pick **Web** as the platform.

Reviewer: ```suggestion
1. Select **Web** as the platform.
```

5. Enter a name for your application (or leave the default value).

> 6. Add the **Base URI** of your application, such as https://grafana.example.com (on-prem)

Reviewer: ```suggestion
1. Add the **Base URI** of your application, such as https://grafana.example.com (for on-prem) or  https://example.grafana.net (for cloud).
```

> https://example.grafana.net (cloud)

Reviewer: ```suggestion
```

7. Enter values for the **Login redirect URI**. Use **Base URI** and append it with `/login/okta`, for example: https://grafana.example.com/login/okta (on-prem)

> https://example.grafana.net/login/okta (cloud)

Reviewer: ```suggestion
```

> 8. Click **Done** to finish creating the Okta application.

Reviewer: ```suggestion
1. Click **Done** to finish creating the Okta application.
```
